src/content/plugins/commons-chunk-plugin.md

```diff
@@ -216,22 +217,6 @@ new webpack.optimize.CommonsChunkPlugin({
 });
 ```
 
-In order to obtain a single CSS file containing your application and vendor CSS, use the following `minChunks` function together with [`ExtractTextPlugin`](/plugins/extract-text-webpack-plugin/):
```

Comment:
Maybe you want to add another PR to describe the way to achieve this with modern plugins/config?

Reply:
Sure, looking into it.